.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
I think this file was not intended to be added in the git history, is it?

Reply:
This is actually intentional since the IDE cannot find the correct Typescript instance otherwise. We can omit it, but then multiple other people will run into the same problem and need to look for a solution. Unless you have other suggestions?